lighthouse-core/gather/gatherers/benchmark-index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const FRGatherer = require('../../fraggle-rock/gather/base-gatherer.js');
+const {getBenchmarkIndex} = require('../../gather/driver/environment.js');
+
+class BenchmarkIndex extends FRGatherer {
+  static symbol = Symbol('BenchmarkIndex');
```

Comment:
We didn't *need* to convert this one, but it was small enough to just include.

Reply:
I'm thinking we shouldn't convert this just yet.

1. We're going to have to special case it again or open warnings up to gatherers to be able to bring the slow CPU warning to fraggle rock.
2. The execution on the page when it might have other things going on during `getArtifact` is going to interfere with the benchmark moreso than when it was on about:blank before the page load. We'll still have that problem for timespans, but navigation is the more important common use of this item for now.
3. Nothing depends on benchmark index so it's just going to disappear until we implement forced inclusion of based artifacts.